tests/rfc003_erc20_htlc.rs

```diff
@@ -553,3 +580,135 @@ fn given_short_zero_secret_htlc_should_not_redeem() {
     );
     assert_eq!(client.token_balance_of(token_contract, bob), U256::from(0));
 }
+
+#[test]
+fn given_invalid_secret_htlc_should_not_redeem_and_print_log() {
+    let docker = Cli::default();
+    let secret_vec = vec![
```

Comment:
This variable should have a better name that makes it clear that it is an INVALID secret!

Reply:
Variable removed.